walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,130 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.*;
+import java.util.stream.Stream; 
+import java.util.stream.Collectors;
 
 public final class FindMeetingQuery {
```

Comment:
Make sure the purpose of your classes are well commented. I recommend Javadoc style.
https://google.github.io/styleguide/javaguide.html#s7-javadoc